### DISCUSSION TITLE
Keyboard support for radio and other toggle components

### DISCUSSION THREAD

**Discussion by bernard-uerm** [quasarframework/quasar#15552] [General - Components / Directives / etc] [answered]
Hi, thank you for your work on Quasar. Just want to know if Quasar supports using keyboard like up and down or left and right keys on selecting or focusing on a specific selection of a radio button component?

Just like this one:
https://codepen.io/hypeJunction/pen/LLBVGX

I want to implement it on my QBtnToggle since I'm working on a survey model.

Thank you in advance.

**Comment by pdanpdan** [comment, marked as answer]
You can use something like this: https://codepen.io/pdanpdan/pen/xxaYmym?editors=1010

**Reply by bernard-uerm** [reply]
Thank you so much for your reply @pdanpdan. So basically, it just focus on the specific selection and not selecting it? Also, is there a built-in functionality on Quasar and not using other libraries. I tried searching other frameworks and it looks like Vuetify has this as a default functionality. Basic HTML also works fine with the keyboard shortcut.

**Reply by pdanpdan** [reply]
I updated the codepen to also select option on focus (check comments in code)
